frontend/components/Forms/SetPrice/SetPrice.js

```diff
@@ -0,0 +1,68 @@
+import { ErrorMessage, Field, Form, Formik } from "formik";
+import React, { useContext } from "react";
+import { FormContext } from "../../Overlay/CreateProject.js";
+import * as yup from "yup";
+import classes from "../../../styles/Forms.module.css";
+
+export default function SetPrice() {
+  const { activeStepIndex, setActiveStepIndex, formData, setFormData } =
+    useContext(FormContext);
+
+  const renderError = (message) => (
+    <p className={classes.errorMessage}>{message}</p>
+  );
+
+  const ValidationSchema = yup.object().shape({
+    SetPrice: yup
+      .number()
+      .label("Price")
+      .positive()
+      .required("A valid price is required")
+      .min(1),
+    minLimit: yup
+      .number()
+      .label("Mint Limit")
+      .positive()
+      .required("A valid number is required")
+      .min(1)
+      .max(100)
+      .integer(),
+  });
+
+  return (
+    <Formik
+      initialValues={{
+        SetPrice: "",
+      }}
+      validationSchema={ValidationSchema}
+      onSubmit={(values) => {
+        const data = { ...formData, ...values };
+        setFormData(data);
+        setActiveStepIndex(activeStepIndex + 1);
+      }}
+    >
+      <Form className={classes.formContainer}>
+        <div className={classes.title}>Set the price!</div>
+        <div className={classes.formContainer}>
+          <label className={classes.labelField}>Price</label>
+          <Field
+            name="SetPrice"
```

Comment:
Related to my comment above on line 16...

```suggestion
            name="setPrice"
```